app/src/main/java/org/mozilla/focus/exceptions/ExceptionsListFragment.kt

```diff
@@ -158,12 +158,15 @@ open class ExceptionsListFragment : BaseSettingsLikeFragment(), CoroutineScope {
         showToolbar(getString(R.string.preference_exceptions))
 
         (binding.exceptionList.adapter as DomainListAdapter).refresh(requireActivity()) {
```

Comment:
You can check with "isAdded" if the fragment is attached.

Reply:
We already use this pattern in numerous places in Fenix, I think we should stick to it.